### PR TITLE
Test on NodeJS 8, 10 and 12

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

**Comment by ryuran** [comment]
Thank you.

@RobLoach Do you remember why you remove node v6 from travis ?
https://github.com/kss-node/kss-node/commit/5c7a4e0c89d87f33ffac633646d82d6d2c8953f6#diff-354f30a63fb0907d4ad57269548329e3

Test seems OK on node6 and it's a supported version (engine in package.json).

File: .travis.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@
 sudo: false
 language: node_js
 node_js:
+  - '6'
   - '8'
-  - '9'
   - '10'
-  - '11'
   - '12'
+  - 'node'
```
Comment (ryuran) [inline .travis.yml]: Why this line ? to test on current stable release ? it should be `'stable'` in this case

Comment (DanielRuf) [reply .travis.yml]: `stable` and `node` are the same for TravisCI afaik

Comment (ryuran) [reply .travis.yml]:
```suggestion
  - 'stable'
```
So I suggest to use `'stable'` because it’s more explicite.

**Comment by RobLoach** [comment]
> @ryuran Do you remember why you remove node v6 from travis ?
5c7a4e0#diff-354f30a63fb0907d4ad57269548329e3

Node.js 6 is no longer supported:
https://nodejs.org/en/about/releases/

**Comment by DanielRuf** [comment]
Right but so far there are still many who use Node 6. Maybe I remove Node 6 when we can be sure that nothing breaks for Node 6 (allowed by the `engines` field).

**Comment by ryuran** [comment]
If kss node stop to support node6, WE should change minimal version in engobe field

**Pull request merged**